tests/test_envs.py

```diff
@@ -1,27 +1,114 @@
+import re
+
 import gym
+import numpy as np
 import pytest
 
-# Import for side-effect of registering environment
+# Unused imports are for side-effect of registering environments
 import imitation.examples.airl_envs  # noqa: F401
 import imitation.examples.model_envs  # noqa: F401
 
 ENV_NAMES = [env_spec.id for env_spec in gym.envs.registration.registry.all()
              if env_spec.id.startswith('imitation/')]
 
+DETERMINISTIC_ENVS = [
+    r"imitation/CliffWorld.*-v0",
```

Comment:
This name is confusing because CliffWorld is not actually deterministic (it just has a deterministic initial state distribution). Maybe call it `DETERMINSTIC_INITIAL_STATE_ENVS` or `DETERMINISTIC_S0_ENVS`?